### ISSUE TITLE
Keyboard navigation shorcut "6" does not work

### ISSUE THREAD

**Issue by jcorporation** [jcorporation/myMPD#1548]
### Discussed in https://github.com/jcorporation/myMPD/discussions/1544

<div type='discussions-op-text'>

<sup>Originally posted by **mgonzalezm** May 19, 2026</sup>
BTW all shortcuts work for me except "6 Show browse playlists"</div>
